### PR TITLE
Remove the Settings and Featured tabs

### PR DESCRIPTION
**Pull request by vegeorgi**
- Remove Settings/Featured folders
- Update README.md
- [Android] Remove icons, leave only labels
- Update screenshots

#346695

### CODE REVIEW

**Comment by manoldonev** [comment]
@vegeorgi Better full TP links http://teampulse.telerik.com/view#item/346695 :)

File: tabs/tabs.component.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ export class TabsComponent implements OnInit {
     * You can find all resources in /App_Resources/os
     *************************************************************/
     getIconSource(icon: string): string {
```
Comment (manoldonev) [inline tabs/tabs.component.ts]: Maybe we should implement a solution similar to the one discussed in https://github.com/NativeScript/template-tab-navigation-ts/pull/27

Comment (vegeorgi) [reply tabs/tabs.component.ts]: I think this is not possible, if we want to keep the template in the way it is. I can make a function to return the whole object depends on the platform and to get rid of the iconSource  equal to empty string. Or to use template tag like here - https://github.com/NativeScript/nativescript-angular/issues/332#issuecomment-239351493 ?

Comment (manoldonev) [reply tabs/tabs.component.ts]: I would go with the template tag.

Comment (vegeorgi) [reply tabs/tabs.component.ts]: I`m ok, if the others are ok too :)

Comment (manoldonev) [reply tabs/tabs.component.ts]: After offline discussion it turned out that the best option is to keep the current implementation.

**Pull request merged**